packages/kit/src/exports/vite/index.js

```diff
@@ -491,9 +518,9 @@ export function set_assets(path) {
 							input,
 							output: {
 								format: 'esm',
-								entryFileNames: ssr ? '[name].js' : `${prefix}/[name]-[hash].js`,
-								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name]-[hash].js`,
-								assetFileNames: `${prefix}/assets/[name]-[hash][extname]`,
+								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].[hash].js`,
+								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].[hash].js`,
+								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
```

Comment:
It's a minor thing, but between these...

```
sverdle-how-to-play-page.js-13bb5e39.js
sverdle-how-to-play-page.js.13bb5e39.js
```

...the second makes more sense to me, because you have four sections of the filename:

```
[route].[ext].[hash].js
```

When they're joined by a hyphen, the extension and the hash become one section which doesn't really make sense